WordPress/Classes/ViewRelated/NUX/LoginWPComViewController.swift

```diff
@@ -0,0 +1,15 @@
+import UIKit
+
+class LoginWPComViewController: SigninWPComViewController {
+    override func dismiss() {
+        self.performSegue(withIdentifier: "showEpilogue", sender: self)
```

Comment:
Ya, you read my mind! (or rather, @natashatherobot's: https://www.natashatherobot.com/protocol-oriented-segue-identifiers-swift/)

I decided to push that off to another PR because this was getting bigger than I liked. What do you think?